### PR TITLE
Feature/service

### PR DESCRIPTION
**Pull request by wollio**
#12 

added the possibility to save services. Similar to products.

### CODE REVIEW

File: stjorna.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (matthiasbaldi) [inline stjorna.iml]: Please add this file or better *.iml to the gitignore

File: server/api/routes_services.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+const prepareAndSaveImage = require('../lib/image_helper.js').prepareAndSaveImage;
+
+module.exports = (router) => {
+    router.route('/v1/services')
+        /**
+         * @api {get} /api/v1/services Get Service List
+         * @apiName GetServicesList
+         * @apiGroup Service
+         * @apiPermission token/apikey
+         * @apiVersion 1.0.0
```
Comment (matthiasbaldi) [inline server/api/routes_services.js]: Please set your API version for the hole PR to 1.1.0 👍

File: client/src/app/_service-form/service-form.component.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<form #serviceForm="ngForm" (ngSubmit)="saveService(serviceForm)">
```
Comment (matthiasbaldi) [inline client/src/app/_service-form/service-form.component.html]: Maybe we have to refactor in the future the form components, we have a lot of duplicated code in here.

Comment (wollio) [reply client/src/app/_service-form/service-form.component.html]: should I create an new issue for that?

Comment (matthiasbaldi) [reply client/src/app/_service-form/service-form.component.html]: If you want 😄 and if you have time to define one.

**Pull request merged**